src/get_window.py

```diff
@@ -16,7 +16,13 @@
     from ewmh import EWMH
     import psutil
     import Xlib
+    import subprocess # Used to interact with kdotool in bash
```

Comment:
This does also add an external dependency for Linux users that has to be installed separately. Might be worth looking into a lib for python that does the same, since kdotool is just a wrapper for another tool that creates JavaScript scripts on the fly for the commands passed in.